frontend/src/components/AddPlayForm.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react' 
+import React, { useState } from 'react' 
```

Comment:
Das `useState` scheint nicht verwendet zu werden
```suggestion
import React from 'react' 
```

Reply:
Danke